docs/content/configuration/virtualserver-and-virtualserverroute-resources.md

```diff
@@ -51,7 +51,7 @@ spec:
 {{% table %}}
 |Field | Description | Type | Required |
 | ---| ---| ---| --- |
-|``host`` | The host (domain name) of the server. Must be a valid subdomain as defined in RFC 1123, such as ``my-app`` or ``hello.example.com``. Wildcard domains like ``*.example.com`` are not allowed.  The ``host`` value needs to be unique among all Ingress and VirtualServer resources. See also [Handling Host and Listener Collisions](/nginx-ingress-controller/configuration/handling-host-and-listener-collisions). | ``string`` | Yes |
+|``host`` | The host (domain name) of the server. Must be a valid subdomain as defined in RFC 1123, such as ``my-app`` or ``hello.example.com``. When using a wildcard domain like ``*.example.com`` the domain must be contained in double quotes.  The ``host`` value needs to be unique among all Ingress and VirtualServer resources. See also [Handling Host and Listener Collisions](/nginx-ingress-controller/configuration/handling-host-and-listener-collisions). | ``string`` | Yes |
```

Comment:
`The ``host`` value needs to be unique`: do we allow two VS resources with same wildcard host? eg. both with `*.something.com`

Reply:
Good point. I'll check if that's possible or not.

Reply:
Tried to deploy two VS resources both with the same wildcard host and got this output:

```
Spec:
  Host:  *.example.com
  Routes:
    Action:
      Pass:  tea
    Path:    /tea
    Action:
      Pass:  coffee
    Path:    /coffee
  Tls:
    Secret:  cafe-secret
  Upstreams:
    Name:     tea
    Port:     80
    Service:  tea-svc
    Name:     coffee
    Port:     80
    Service:  coffee-svc
Status:
  Message:  Host is taken by another resource
  Reason:   Rejected
  State:    Warning
Events:
  Type     Reason    Age   From                      Message
  ----     ------    ----  ----                      -------
  Warning  Rejected  13s   nginx-ingress-controller  Host is taken by another resource
```